programs/management/commands/remove_programs.py

```diff
@@ -0,0 +1,61 @@
+from django.core.management.base import BaseCommand
+from programs.models import (
+    Program,
+    Navigator,
+)
+from translations.models import Translation
+
+
+class Command(BaseCommand):
+    help = (
+        'Remove all programs and navigators and their translations'
+    )
+
+    def handle(self, *args, **options):
+        program_translated_fields = (
+            'description_short',
+            'name',
+            'description',
+            'learn_more_link',
+            'apply_button_link',
+            'value_type',
+            'estimated_delivery_time',
+            'estimated_application_time',
+            'category',
+            'warning',
+        )
+
+        navigator_translated_fields = (
+            'name',
+            'email',
+            'assistance_link',
+            'description',
+        )
+
+        navigators = Navigator.objects.all()
+        programs = Program.objects.all()
+
+        for navigator in navigators:
+            translations_to_delete = []
+            for field in navigator_translated_fields:
+                translations_to_delete.extend(
+                    Translation.objects.filter(
+                        label__startswith=f'navigator.{navigator.name}_{navigator.id}-{field}')
```

Comment:
Does this work? That way, if the label changes, we don't have to worry about changing this script.
```suggestion
                    getattr(navigator, field)
```